lib/gds_api/test_helpers/local_links_manager.rb

```diff
@@ -98,13 +98,13 @@ def convert_to_query_string_params(parameters)
         parameters
       end
 
-      def stub_local_links_manager_has_a_local_authority(authority_slug, local_custodian_code: nil)
+      def stub_local_links_manager_has_a_local_authority(authority_slug, country_name: "England", local_custodian_code: nil)
```

Comment:
Commit message slightly inaccurate - suggests that `local_custodian_code` is being added to the above method, when it's being added to the `stub_local_links_manager_has_a_local_authority_without_homepage` method. Could either split these additions out into two commits, or tweak the message.